### PR TITLE
Button and Contributor styling

### PR DESCRIPTION
**Pull request by lortza**
Resources page buttons: turned the edit and trash icons into buttons and floated them to the right to make them more consistent looking and easier to click.

About page: added styling and images to the project contributors

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,51 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+## schema
+  # create_table "categories"
+    # t.string   "name"
+    # t.text     "description"
+
+Category.create!([
```
Comment (siakaramalegos) [inline db/seeds.rb]: So, there is already a `rake db:populate` task that populates fake category and resource data.  Also, production is already live so not sure the usefulness of a seed file.  Everything else sounds great though of course I haven't seen it yet.  :)  If you want some of these categories in production, just make them on the site.

**Comment by siakaramalegos** [comment]
Hey girl - if you just take out the seeds change, I can pull everything else in.

**Comment by lortza** [comment]
@siakaramalegos, I just removed the seeds, so you should be good to go. :)

**Comment by siakaramalegos** [comment]
Love the new contributors page!  Great job.  Deploying now so hopefully will be up in a sec. I added a 10px margin to the resource-icons so they link up better with the resource titles.  I also fixed some of the column responsiveness on the categories page.

**Comment by lortza** [comment]
Nice! I'm glad I could finally contribute.

Anne Richardson
portfolio http://localflavormarketing.com/ | linkedin
https://www.linkedin.com/in/localflavor | github
https://github.com/lortza | side projects
https://www.thegamecrafter.com/designers/anne-richardson | travel blahg
http://viajaryamar.com/

**Pull request merged**